### PR TITLE
Prom

### PR DESCRIPTION
**Pull request by ocar1053**
update readme and add request limit  in  jperstore-backend-service yaml

### CODE REVIEW

**Comment by ocar1053** [comment]
thanks for you update

**Comment by ocar1053** [comment]
add some pic

**Review by Lanznx — COMMENTED** [review]
有圖真讚！

File: README.md [hunk 1]
```diff
@@ -1,9 +1,61 @@
...
+4. set load balancer to expose service public ip
+```
+kubectl expose deployment jpetstore-backend-deployment --type="LoadBalancer" -n jpetstore
+```
+5. Enter public ip in browser to access jpetstore
+
+![image](https://github.com/ocar1053/JPetStore/assets/64206644/6ecac045-92fb-4573-a93f-0a39d3d381e7)
+
 
 ## Prometheus 以及 Grafana 架設說明 (頤賢）
```
Comment (Lanznx) [inline README.md:35]: 可以把名字拿掉嗎

**Review by Lanznx — COMMENTED** [review]
LGTM

**Pull request merged**